source/patterns/@aws-solutions-constructs/aws-lambda-sagemakerendpoint/lib/index.ts

```diff
@@ -100,7 +93,7 @@ export class LambdaToSagemakerEndpoint extends cdk.Construct {
    * @param {cdk.App} scope - represents the scope for all the resources.
    * @param {string} id - this is a scope-unique id.
    * @param {LambdaToSagemakerEndpointProps} props - user provided props for the construct.
-   * @since 1.76.0
+   * @since 1.86.0
```

Comment:
1.87.0 is already out. This should be 1.87.1 (we'll hold off that release for this).